invokeai/app/services/restoration_services.py

```diff
@@ -62,13 +64,16 @@ def upscale_and_reconstruct(
                                     ">> GFPGAN not found. Face restoration is disabled."
                                 )
                             else:
-                                image = self.gfpgan.process(image, strength, seed)
+                                print(">> Restoring with: GFPGAN")
```

Comment:
Didn't @lstein set up logging recently? We should be avoiding print in favor of logging, if so.

Reply:
That is not merged yet. Can port it over then.